scripts/integration-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+INTEGRATION_TEST=$1
+SRC_PATH=$2
+RELATIVE_PATH="${3:-../../}" # set to third arg if defined, otherwise use ../../
```

Comment:
This can (but is not currently) account for deeply nested integration/e2e tests that need to modify the relative path.